### DISCUSSION TITLE
Live Updating Datatable

### DISCUSSION THREAD

**Discussion by CovertError** [rappasoft/laravel-livewire-tables#615] [Q&A] [answered]
the idea is to be able to see records after them being created without having to refresh the page I would love to create a PR for that but I'm new to developing such things so I might need some assistance

**Comment by rappasoft** [comment, marked as answer]
You have 2 options:

- https://rappasoft.com/docs/laravel-livewire-tables/v1/customizing/enabling-polling-to-refresh
- Fire this event: https://github.com/rappasoft/laravel-livewire-tables/blob/master/src/DataTableComponent.php#L113

**Reply by CovertError** [reply]
I was thinking of something that would just add a new row whenever a new record gets created without a refresh and maybe highlight it or something like that

**Reply by rappasoft** [reply]
`public $refresh = 1000;`

That will re-render the table every second. But keep in mind that makes a call to the server every second so think about how often you need to check.

**Reply by CovertError** [reply]
okay thanks so much for the suggestion
